docs/graphql/manual/api-reference/schema-metadata-api/manage-metadata.rst

```diff
@@ -77,9 +77,28 @@ table using ``psql`` and this column should now be added to the GraphQL schema.
 
    {
        "type" : "reload_metadata",
-       "args": {}
+       "args": {
+           "reload_remote_schemas": true
+       }
    }
 
+.. _reload_metadata_args_syntax:
+
+Args syntax
+^^^^^^^^^^^
+
+.. list-table::
+   :header-rows: 1
+
+   * - Key
+     - Required
+     - Schema
+     - Description
+   * - reload_remote_schemas
+     - false
+     - Boolean
+     - If set to ``true``, all remote schemas' (including inconsistent ones) cached GraphQL schemas are refreshed (default: ``false``)
```

Comment:
@rakeshkky Why do we not reload the remote schemas when we do `reload_metadata`? Any particular reason?

Reply:
@0x777 https://github.com/hasura/graphql-engine/issues/3792#issue-556619670